core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
```

Comment:
There might be an issue with merge conflicts later on. I'd suggest that we stagger the merge of the PR's. So if @shaz13 thinks that his PR can be merged first or vice versa. Also, PR contents ideally shouldn't have duplicate code.